Resources/Prototypes/_NF/Events/nf_bluespace_grids_events.yml

```
@@ -12,7 +12,7 @@
     weight: 5
     duration: 1350
     maxDuration: 1560
-    reoccurrenceDelay: 480 # 8 hours
+    reoccurrenceDelay: 240 # 4 hours
```

```suggestion
    reoccurrenceDelay: 360 # 6 hours
```
Once in a six hour shift.